packages/gp2-components/src/organisms/OpenQuestionsModal.tsx

```diff
@@ -0,0 +1,137 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { Button, pixels, TextArea } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { ComponentProps, useState } from 'react';
+import { addIcon, binIcon } from '../icons';
+import { mobileQuery } from '../layout';
+import EditUserModal from './EditUserModal';
+
+const { rem } = pixels;
+
+const containerStyles = css({
+  [mobileQuery]: {
+    display: 'unset',
+  },
+  display: 'flex',
+  paddingBottom: rem(8),
+  flexDirection: 'column',
+});
+
+const rowStyles = css({
+  marginBottom: rem(16),
+  padding: `${rem(16)} 0`,
+  ':last-child': {
+    borderBottom: 'none',
+    marginBottom: 0,
+    paddingBottom: 0,
+  },
+});
+
+const headerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+});
+
+const buttonStyles = css({ margin: 0 });
+
+const addButtonStyles = css({
+  width: 'fit-content',
+  [mobileQuery]: {
+    width: '100%',
+  },
+});
+
+const optional = '(optional)';
+
+type OpenQuestionsModalProps = Pick<gp2Model.UserResponse, 'questions'> &
+  Pick<ComponentProps<typeof EditUserModal>, 'backHref'> & {
+    onSave: (userData: gp2Model.UserPatchRequest) => Promise<void>;
+  };
+
+const OpenQuestionsModal: React.FC<OpenQuestionsModalProps> = ({
+  questions,
+  backHref,
+  onSave,
+}) => {
+  const [newQuestions, setQuestions] = useState(
+    !questions.length ? [''] : questions,
+  );
+  const checkDirty = () =>
+    newQuestions.some((question, index) => question !== questions[index]);
```

Comment:
I believe this means that if you open the modal and just try to close it without changing anything it will show the warning message to prevent changes loss when nothing was changed. Maybe we can improve this.